socorro/cron/crontabber_app.py

```diff
@@ -1093,5 +1093,8 @@ def sentrytest(self):
     'socorro.cron.jobs.update_signatures.UpdateSignaturesCronApp|1h',
 
     # Dependency checking
-    'socorro.cron.jobs.monitoring.DependencySecurityCheckCronApp|1d',
+    'socorro.cron.jobs.monitoring.DependencySecurityCheckCronApp|1d|05:00',
+
+    # Processed crash verification
+    'socorro.cron.jobs.verify_processed.VerifyProcessedCronApp|1d|04:00',
```

Comment:
On my local machine, this takes 7 minutes to run. That's entirely iterating through the 4096 entropy possibilities and doing a listing for each one. I suspect it'll take longer on stage and even longer in prod, but I don't think it'll take more than an hour.